app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/health/AppTPCPUMonitor.kt

```diff
@@ -46,17 +43,13 @@ class AppTPCPUMonitor @Inject constructor(
     }
 
     override fun onVpnStarted(coroutineScope: CoroutineScope) {
-        if (appTpFeatureConfig.isEnabled(AppTpSetting.CPUMonitoring)) {
-            logcat { "AppTpSetting.CPUMonitoring is enabled, starting monitoring" }
-            val work = PeriodicWorkRequestBuilder<CPUMonitorWorker>(4, TimeUnit.HOURS)
-                .boundToVpnProcess(appBuildConfig.applicationId) // this worker is executed in the :vpn process
-                .setInitialDelay(10, TimeUnit.MINUTES) // let the CPU usage settle after VPN restart
-                .build()
+        logcat { "AppTpSetting.CPUMonitoring is enabled, starting monitoring" }
```

Comment:
@aitorvs is this something we only want to run for AppTP only? (Assuming from the name)

Reply:
Name is legacy, originally added for AppTP. But I think we want it for the entire VPN regardless.
We can revisit later if needed.